plugins/web-plugin/src/test/java/pagefactory/pages/webelements/AbstractPage.java

```diff
@@ -24,19 +24,21 @@
     @ElementTitle("HomeRedirect")
     @FindBy(xpath = "//a[text()='Home']")
     private WebElement homeButtonWithRedirect;
-    
+
     public AbstractPage(WebDriver driver) {
         super(driver);
     }
-    
+
     @ActionTitle("go to page")
     public void goToPage(String pageName) {
         switch (pageName.toLowerCase()) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Add a default case to this switch. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ASwitchLastCaseIsDefaultCheck)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this "switch" statement by "if" statements to increase readability. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1301)